bitnami/kafka/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kafka
   - https://kafka.apache.org/
-version: 19.0.1
+version: 19.0.2
```

Comment:
Please bump the chart to a new minor version, you are adding new functionalities.

```
19.1.0
```